core/pom.xml

```diff
@@ -11,6 +11,11 @@
   <properties>
     <guava.version>20.0</guava.version>
     <gwt.version>2.8.0</gwt.version>
+    <junit.version>4.12</junit.version>
+    <jsr305.version>3.0.1</jsr305.version>
+    <auto-value.version>1.4.1</auto-value.version>
+    <compile-testing.version>0.10</compile-testing.version>
```

Comment:
Yup, we build for 1.6 even. https://github.com/google/truth/blob/cb03a71134ffb9941a82940b72f17fbfec7b9825/core/pom.xml#L87